scripts/tests/run_test_suite.py

```diff
@@ -165,25 +167,31 @@ def cmd_list(context):
     help='Number of iterations to run')
 @click.option(
     '--all-clusters-app',
-    default=FindBinaryPath('chip-all-clusters-app'),
     help='what all clusters app to use')
 @click.option(
     '--door-lock-app',
-    default=FindBinaryPath('chip-door-lock-app'),
     help='what door lock app to use')
 @click.option(
     '--tv-app',
-    default=FindBinaryPath('chip-tv-app'),
     help='what tv app to use')
 @click.option(
     '--pics-file',
     type=click.Path(exists=True),
-    default=FindBinaryPath("ci-pics-values"),
+    default="src/app/tests/suites/certification/ci-pics-values",
     help='PICS file to use for test runs.')
 @click.pass_context
 def cmd_run(context, iterations, all_clusters_app, door_lock_app, tv_app, pics_file):
     runner = chiptest.runner.Runner()
 
+    if (all_clusters_app is None):
```

Comment:
brackets not necessary.

Also wonder (not for this pr) if we should not try to find applications that we never use as not all tests require tv or door lock for example. Requires larger changes though.